tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -45,3 +46,21 @@ def disable_non_trt_optimizers_in_rewriter_config(rewriter_config):
   rewriter_config.remapping = off
   rewriter_config.scoped_allocator_optimization = off
   rewriter_config.shape_optimization = off
+
+
+def IsLinkedTensorRTVersionGreaterEqual(major, minor=0, patch=0):
+  ver = _pywrap_py_utils.get_linked_tensorrt_version()
+  assert isinstance(ver, tuple)
+  assert len(ver) == 3
+
+  return ver[0] > major or (ver[0] == major and ver[1] > minor) or (
+      ver[0] == major and ver[1] == minor and ver[2] >= patch)
```

Comment:
Let's put this into a local function for sharing with the next function.

Reply:
Fixed